### PR TITLE
chore: [Design System V2] Update buttons in the bottom bar

### PR DESCRIPTION
**Pull request by hetunandu**
## Description

Fixes the buttons in the bottom bar right corner with the new DS buttons

https://www.loom.com/share/758ca4a30aa343c1996314d375fc86cc

### CODE REVIEW

**Comment by rohitagarwal88** [comment]
@hetunandu : Changes look good, only change is to dedup dropdown with Menu component
Screenshot 2023-04-17 at 12 20 52 PM

File: app/client/src/components/editorComponents/Debugger/index.tsx [hunk 1]
```diff
@@ -113,30 +58,18 @@ export function DebuggerTrigger() {
...
+        startIcon={totalMessageCount ? "close-circle" : "close-circle-line"}
       >
-        <Button
-          isIconButton
-          kind="tertiary"
-          onClick={onClick}
-          size="md"
-          startIcon={totalMessageCount ? "close-circle" : "close-circle-line"}
-        />
-        <div className="debugger-count t--debugger-count">
```
Comment (rohitagarwal88) [inline app/client/src/components/editorComponents/Debugger/index.tsx:135]: @hetunandu : Please do not remove `t--debugger` class. It is used in the cypress test.

File: app/client/src/pages/Editor/HelpButton.tsx [hunk 2]
```diff
@@ -54,21 +27,17 @@ function HelpButton() {
...
-      disabled={isHelpOpen}
-      hoverOpenDelay={TOOLTIP_HOVER_ON_DELAY}
-      modifiers={{
-        preventOverflow: { enabled: true },
-      }}
-      position={"bottom"}
+      // disabled={isHelpOpen}
+      placement="bottom"
     >
       <Popover
```
Comment (rohitagarwal88) [inline app/client/src/pages/Editor/HelpButton.tsx:42]: @hetunandu : Can popover with our ADS 2.0 menu component? 

https://design-system.appsmith.com/?path=/story/design-system-menu--menu-story

**Review by rohitagarwal88 — CHANGES_REQUESTED** [review]

File: app/client/src/pages/Editor/HelpButton.tsx (hunk 2 above)
Comment (hetunandu) [reply app/client/src/pages/Editor/HelpButton.tsx]: Updated

File: app/client/src/components/editorComponents/Debugger/index.tsx (hunk 1 above)
Comment (hetunandu) [reply app/client/src/components/editorComponents/Debugger/index.tsx:135]: Not removed. Just moved it to another place

File: app/client/src/components/editorComponents/Debugger/index.tsx [hunk 3]
```diff
@@ -113,30 +58,18 @@ export function DebuggerTrigger() {
...
 
   return (
-    <TooltipComponent
-      content={tooltipContent}
-      modifiers={{
-        preventOverflow: { enabled: true },
-      }}
-    >
-      <TriggerContainer
-        className="t--debugger"
```
Comment (rohitagarwal88) [inline app/client/src/components/editorComponents/Debugger/index.tsx:123]: Sry @hetunandu, tagged the wrong code,  Is this in another file? 
https://github.com/appsmithorg/appsmith/pull/22356/files/a68e462d63bbc345b24c19d4917bd13ab982d36c#r1168257182

**Comment by albinAppsmith** [comment]
I see in release the bottom count button is red and in DP it is gray... Is it okay?
Screenshot 2023-04-21 at 12 27 28 PM

File: app/client/src/components/editorComponents/Debugger/index.tsx [hunk 4]
```diff
@@ -113,30 +58,18 @@ export function DebuggerTrigger() {
...
-        errorCount={messageCounters.errors}
+    <Tooltip content={tooltipContent}>
+      <Button
+        className="t--debugger-count"
+        color={"red"}
```
Comment (albinAppsmith) [inline app/client/src/components/editorComponents/Debugger/index.tsx:64]: Button do not have a color prop

File: app/client/src/pages/Editor/HelpButton.tsx [hunk 5]
```diff
@@ -1,100 +1,134 @@
...
-  font-weight: 400;
-  padding: 9px 16px;
-  border-left: 1px solid #e7e7e7;
-  cursor: pointer;
-  ${BottomBarCTAStyles};
-  -webkit-user-select: none; /* Safari */
-  -ms-user-select: none; /* IE 10 and IE 11 */
-  user-select: none; /* Standard syntax */
+  justify-content: space-between;
+  border-top: 1px solid ${Colors.ALTO};
```
Comment (albinAppsmith) [inline app/client/src/pages/Editor/HelpButton.tsx:32]: Please update the border color with `var(--ads-v2-color-border)`

File: app/client/src/pages/Editor/HelpButton.tsx [hunk 6]
```diff
@@ -1,100 +1,134 @@
...
+  padding: 5px 10px;
+  height: 30px;
+  color: ${Colors.DOVE_GRAY2};
```
Comment (albinAppsmith) [inline app/client/src/pages/Editor/HelpButton.tsx:35]: Please update color with `var(--ads-v2-color-fg)`

**Review by albinAppsmith — APPROVED** [review]

**Comment by laveena-en** [comment]
1. Tooltip is missing for Help 
image
2. Go-to icon to be seen on hover and not as a static icon 
image
3. Tooltip isn’t getting dismissed https://www.loom.com/share/698bd5a33ec349a982ccd592044ef6f8
4. Pointer cursor required on hovering over the update icon.
image
5. Ellipses and a tooltip is required when the branch name is very long https://www.loom.com/share/1758b634e9a14d7da277a717e17845c5
6. Heading colours are not as per ADS2.0. Added more details here: https://www.notion.so/appsmith/Heading-colours-are-not-as-per-ADS2-0-2d695457152f40af93a7ad40c2a762e5?pvs=4
7. Size and padding of the update icon looks off, can we add in some padding and make the size consistent 
image

**Pull request merged**